src/CommandLine/OptionAttribute.cs

```diff
@@ -114,5 +116,14 @@ public string Group
             get { return group; }
             set { group = value; }
         }
+
+        /// <summary>
+        /// When applied to an int property, turns that property into a count of how many times a boolean flag was applied (e.g., -vvv would become 3)
+        /// </summary>
+        public bool FlagCounter

```

Comment:
Provide Test case for testing FlagCounter.
It seems it's not working and raise error when trying to use a commandline: '-vvv'  or "-v -v -v" with int property
```cs
 class Options
        {
            [Option('v',FlagCounter =true)]
            public int Visible { get; set; } //expect to be 3
        }
```

Reply:
I ended up removing the FlagCounter feature. The work I've done on it so far, including a test case, is in the https://github.com/rmunn/commandline/tree/wip/flagCounter branch of my repo. As suggested by the `wip` in the name, it's a work-in-progress that is not yet ready for a PR or merge, and I may squash or force-push that branch at any time.

But with that feature removed from this PR, this particular code review is resolved.